app/src/androidTest/java/com/android/bookswap/ui/map/MapScreenTest.kt

```diff
@@ -130,7 +161,11 @@ class MapScreenTest {
 
   @Test
   fun draggableMenu_canBeDraggedVertically() {
-    composeTestRule.setContent { MapScreen(user, user[0]) }
```

Comment:
The title of the test is a bit inconsistent here, you seem to use camelCase everywhere but one word in this test title. Though I must say that given everything that could either change or have gone wrong, this one is practically insignificant.